registry/storage/driver/azure/azure.go

```diff
@@ -23,11 +24,16 @@ import (
 const driverName = "azure"
 
 const (
-	paramAccountName = "accountname"
-	paramAccountKey  = "accountkey"
-	paramContainer   = "container"
-	paramRealm       = "realm"
-	maxChunkSize     = 4 * 1024 * 1024
+	paramAccountName  = "accountname"
+	paramAccountKey   = "accountkey"
+	paramContainer    = "container"
+	paramRealm        = "realm"
+	paramMaxIdleConns = "maxidleconns"
+	maxChunkSize      = 4 * 1024 * 1024
+)
+
+var (
```

Comment:
Oops. This should be part of the `const` block above.

Reply:
It's not a const. It's configurable by the configuration file. The default is 100 but it gives user a way to change based on their load.

Reply:
This is a package global. It should not be changeable by the configuration file.

Reply:
Thanks. I updated the RP. Now the global is removed, and a const default is added. Driver initialization will read configuration. If no configuration, the default will be used; if there is, use the configuration value.